datajunction-clients/python/datajunction/compile.py

```diff
@@ -409,11 +419,11 @@ class Project(BaseModel):
     mode: NodeMode = NodeMode.PUBLISHED
 
     @classmethod
-    def load_current(cls):
+    def load_current(cls, directory: Optional[str] = None):
```

Comment:
Added this largely because it isn't that easy to switch directories when running from a notebook. The only way to successfully do that is to restart Jupyter from a different directory, so seemed more convenient to have the ability to select which directory to load when invoking the client from a notebook.

Reply:
I imagined that would a different method we would eventually add. Load current project is meant to detect which project you currently are in and works if you are deeply nested within a sub-directory. What do you think of just making another method called `load_project(...)` that takes a directory path as an argument?

Reply:
@samredai agreed, that definitely makes more sense! I just added a new `load` class method